src/Database/Driver.php

```diff
@@ -258,7 +258,7 @@ public function schemaValue($value)
             is_numeric($value) && strpos($value, ',') === false &&
             $value[0] !== '0' && strpos($value, 'e') === false)
         ) {
-            return $value;
+            return (string)$value;
```

Comment:
Ah sorry. I misread the code. If users use this method to pass bindValue(), then that is users' fault.